napari/_vispy/vendored/volume.py

```diff
@@ -396,7 +396,7 @@ class VolumeVisual(Visual):
         The contrast limits. The values in the volume are mapped to
         black and white corresponding to these values. Default maps
         between min and max.
-    method : {'mip', 'translucent', 'additive', 'iso'}
+    method : {'mip', 'translucent', 'additive', 'iso', 'avg'}
```

Comment:
```suggestion
    method : {'mip', 'translucent', 'additive', 'iso'}
```

I'd revert this, and try and keep changes in the `_vispy/volume.py` file and not both that and `_vispy/vendored/volume.py` (which we want to be able to drop when the next vispy release comes out)